app/main/views/g_cloud.py

```diff
@@ -544,6 +544,62 @@ def end_search(framework_framework, project_id):
     )
 
 
+@direct_award.route(
+    '/<string:framework_framework>/projects/<int:project_id>/did-you-award-contract',
+    methods=['GET', 'POST']
+)
+def did_you_award_contract(framework_framework, project_id):
+    all_frameworks = data_api_client.find_frameworks().get('frameworks')
+    framework = framework_helpers.get_latest_live_framework(all_frameworks, framework_framework)
+
+    # Get the requested Direct Award Project.
+    project = data_api_client.get_direct_award_project(project_id=project_id)['project']
+
+    form = DidYouAwardAContractForm()
+    form.did_you_award_a_contract.options = [{"label": label, "value": value}
+                                             for value, label in form.did_you_award_a_contract.choices]
+
+    if form.validate_on_submit():
+        if form.did_you_award_a_contract.data == 'still-assessing':
+            return redirect(url_for('.view_project',
+                                    framework_framework=framework_framework,
+                                    project_id=project_id))
+        elif form.did_you_award_a_contract.data == 'yes':
+            return redirect(url_for('.which_service_won_contract',
+                                    framework_framework=framework_framework,
+                                    project_id=project_id))
+        elif form.did_you_award_a_contract.data == 'no':
+            return redirect(url_for('.why_didnt_you_award_contract',
+                                    framework_framework=framework_framework,
+                                    project_id=project_id))
+        else:
+            # this else should never be reached
+            abort(500, "Unexpected answer to 'Did you award contract?'")
+
+    return render_template(
+        'direct-award/did-you-award-contract.html',
+        form=form,
+        project=project,
+        framework=framework
+    ), 200 if not form.errors else 400
+
+
+@direct_award.route(
+    '/<string:framework_framework>/projects/<int:project_id>/which-service-won-contract',
+    methods=['GET']
+)
+def which_service_won_contract(framework_framework, project_id):
+    pass
+
+
+@direct_award.route(
+    '/<string:framework_framework>/projects/<int:project_id>/why-didnt-you-award-contract',
```

Comment:
So I know it's kinda late to be mentioning this, but I really don't like URI's that look like they are missing punctuation. `.../non-award-reason` perhaps? cc @karlchillmaid

Reply:
Response from @karlchillmaid on Slack

> @george.lund @dilwoar.hussain re the URL question on GitHub — the URLs follow the standards for GOV.UK: https://insidegovuk.blog.gov.uk/url-standards-for-gov-uk/
The guidelines don't say anything about contractions, but when we discussed this at content crit we found other examples that do, eg. https://www.gov.uk/expenses-if-youre-self-employed
I agree that contractions look a bit funny without the apostrophe, but I think they're still readable, and it's probably more important to keep the URLs as close as possible to the page title and use logical, consistent, predictable patterns for URLs.